### PR TITLE
Doc 851 log-collector script instructions

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-851 
suggested by Vikram Moule in #ask-k8s channel 8/12/2021

Staging link: https://docs.redis.com/staging/DOC-851/platforms/kubernetes/tasks/run-log-collector-script/

### CODE REVIEW

File: content/platforms/kubernetes/tasks/run-log-collector-script.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+Title: Run the log collector script
+linkTitle: Run log_collector.py
```
Comment (rrelledge) [inline content/platforms/kubernetes/tasks/run-log-collector-script.md:3]:
```suggestion
linkTitle: Run log_collector.py
description: Run the log collector script to package relevant logs into a tar.gz file to send to Redis Support for help troubleshooting your Kubernetes environment.
```

**Review by rrelledge — APPROVED** [review]
Looks good. I left a tentative suggestion to add a short description so that its parent page Tasks won't autogenerate a weird description from the entire page.

One last idea: should there be a sentence that describes why a user would want to use log_collector.py on this new page as well, such as for help troubleshooting?

**Comment by laurentdroin** [comment]
>  Pass -n parameter to run on the current namespace.

You use the -n flag to run the script against a namespace that is NOT the current namespace. By default (i.e. when you don't use -n, the script runs against the current namespace).

File: content/platforms/kubernetes/tasks/run-log-collector-script.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+description: Run the log collector script to package relevant logs into a tar.gz file to send to Redis Support for help troubleshooting your Kubernetes environment.
+weight: 89
+alwaysopen: false
+categories: ["Platforms"]
+aliases: 
+---
+
+The Redis Enterprise cluster (REC) log collector script ([`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py)) creates and fills a directory with the relevant logs for your environment.
```
Comment (kaitlynmichael) [inline content/platforms/kubernetes/tasks/run-log-collector-script.md:11]:
```suggestion
The Redis Enterprise cluster (REC) log collector script ([`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py)) creates and fills a directory with the relevant logs for your environment. These logs will help the support team with troubleshooting. 
```

File: content/platforms/kubernetes/tasks/run-log-collector-script.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+alwaysopen: false
+categories: ["Platforms"]
+aliases: 
+---
+
+The Redis Enterprise cluster (REC) log collector script ([`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py)) creates and fills a directory with the relevant logs for your environment. These logs will help the support team with troubleshooting. 
+
+1. Download the [`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py) file.
+2. Run the script on the system that runs your `kubectl` or `oc` commands.
+    - Pass `-n` parameter to run on the current namespace.
```
Comment (kaitlynmichael) [inline content/platforms/kubernetes/tasks/run-log-collector-script.md:15]:
```suggestion
    - Pass `-n` parameter to run on a different namespace than the one you are currently on.
```

**Pull request merged**